portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -113,39 +118,109 @@ function handleAuthorizationError(error) {
  * message on the UI.
  */
 function handleExportError(reason) {
-  var $sheetsMessage = document.getElementById('sheets-message');
-  $sheetsMessage.removeClass('d-none')
+  $('#sheets-message')
+      .removeClass('d-none')
       .text('Error: ' + reason.result.error.message)
       .show();
 }
 
+/**
+ * Helps the SheetsApi requests easily create the values it needs for a
+ * spreadsheet.
+ */
+function makeSheetsValuesFromScheduledTasks(scheduledTasks, values) {
+  for (index = 0; index < scheduledTasks.length; index++) {
+    values.push(singleScheduledTaskToSheetsArray(scheduledTasks[index]));
+  }
+}
+
+/**
+ * This takes in a single JSON object of a ScheduledTask (Java Class) and
+ * turns it into the correct format for the Sheets API
+ */
+function singleScheduledTaskToSheetsArray(scheduledTask) {
```

Comment:
Consider adding unit test for this method.

Reply:
Will include that in the next PR. Added a TODO.